src/ui/screens/Parachains/AuctionsScreen.tsx

```diff
@@ -1,239 +1,114 @@
-        return [data.endBlock, bestNumber];
-      } else if (data.endingPeriod) {
-        return [data.endingPeriod, bestNumber.sub(data.endBlock)];
-      }
-    }
-    return [BN_ZERO, BN_ZERO];
-  }
-  const [endingAt, currentPosition] = getEndingPeriodValues();
-  const remainingPercent = endingAt.isZero() ? 0 : currentPosition.muln(10000).div(endingAt).toNumber() / 10000;
-
-  if (auctionIsError || winningDataIsError || issuanceIsError || endpointsIsError) {
-    return <ErrorView />;
+  const {data: auction, loading} = useAuctionsSummary();
+  const snackbar = useSnackbar();
+
+  if (loading && !auction) {
+    return (
+      <SafeView edges={noTopEdges}>
+        <LoadingView />
+      </SafeView>
+    );
   }
 
-  if (!data || auctionInfoIsLoading) {
-    return <LoadingView />;
+  if (!auction) {
+    return (
+      <SafeView edges={noTopEdges}>
+        <EmptyView />
+      </SafeView>
+    );
   }
 
-  const tableIsLoading = auctionInfoIsLoading || winningDataIsLoading || issuanceIsLoading || endpointsIsLoading;
-  const items: DataTableProps['items'] =
-    winningData?.map((wd) => {
-      return {
-        blockNumber: String(formatNumber(wd.blockNumber)),
-        projectID: String(formatNumber(wd.winners[0]?.paraId)),
-        projectName: endpoints?.find((e) => e.paraId === wd.winners[0]?.paraId.toNumber())?.text?.toString() || '',
-        value: String(formatBalance(wd.total)),
-      };
-    }) ?? [];
+  const {auctionsInfo, latestAuction} = auction;
+  const {winningBid, leasePeriod, endingPeriod, raised, raisedPercent} = latestAuction;
 
-  const isActive = Boolean(data.leasePeriod);
+  const copyToClipboard = () => {
+    if (winningBid?.blockNumber) {
+      Clipboard.setString(winningBid.blockNumber);
+      snackbar('Blocknumber copied to clipboard!');
+    }
+  };
 
   return (
-    <Layout style={globalStyles.flex}>
-      <ScrollView>
-        <Header
-          count={formatNumber(data.numAuctions) ?? 0}
-          active={isActive}
-          firstLast={{
-            first: formatNumber(data.leasePeriod),
-            last: formatNumber(data.leasePeriod?.add((data.leasePeriodsPerSlot as u32) ?? BN_ONE).isub(BN_ONE)),
-          }}
-          stats={{raised: formatBalance(raised, {isShort: true}) ?? '', raisedPercent}}
-          duration={{endingAt, remaining: endingAt.sub(currentPosition), remainingPercent}}
-        />
-        {isActive ? (
-          <DataTableComp items={items} loading={tableIsLoading} />
-        ) : (
-          <View style={globalStyles.centeredContainer}>
-            <Caption>The auction is not active.</Caption>
+    <SafeView edges={noTopEdges}>
+      <Padder scale={1} />
+      <Card style={styles.container}>
+        <Card.Content>
+          <View style={styles.itemRow}>
+            <StatInfoBlock title="Auctions">{auctionsInfo.numAuctions}</StatInfoBlock>
+            <StatInfoBlock title="Active">{auctionsInfo.active ? 'Yes' : 'No'}</StatInfoBlock>
```

Comment:
Providing text color for Yes and  No would be useful

Reply:
Let's focus on displaying the information in a simple way for now, since we will be having proper designs soon.